src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -85,22 +93,23 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 showProgress: true,
                 c =>
                 {
-                    var cancellationToken = context.OperationContext.UserCancellationToken;
-
-                    using (var transaction = new CaretPreservingEditTransaction(
-                        EditorFeaturesResources.Formatting, args.TextView, _undoHistoryRegistry, _editorOperationsFactoryService))
+                    using (Logger.LogBlock(FunctionId.FormatDocument, CodeCleanupLogMessage.Create(docOptions), cancellationToken))
```

Comment:
everything inside of Wait should use "c" instead of cancellation token from context.

Reply:
c.CancellationToken I mean